src/TableBody.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { forwardRef } from 'react';
+import createStyles from './createStyles';
+import { PropsFromStyles } from './types';
+
+const useStyles = createStyles(({ css, theme }) => ({
+  // table body base styles
+  root: css``,
+}));
+
+type TableBodyProps = JSX.IntrinsicElements['tbody'];
+
+interface Props extends PropsFromStyles<typeof useStyles>, TableBodyProps {}
+
+const TableBody = forwardRef((props: Props, ref: React.Ref<any>) => {
+  const { Root, styles, ...restOfProps } = useStyles(props, 'tbody');
+  return <Root ref={ref} {...restOfProps} />;
+});
+
+TableBody.displayName = 'TableBody';
```

Comment:
This is interesting. This component doesn't do anything so I'm tempted to say we should remove it but I understand wanting it to keep for for consistency's sake.

At the very least, we can change the implementation to be `const TableBody = 'tbody';` lol

Let me know what you think

```suggestion
const TableBody = 'tbody';
```

Reply:
Lol will do!